contracts/system_contracts/kip113/KIP113.sol

```diff
@@ -0,0 +1,118 @@
+// Copyright 2023 The klaytn Authors
+// This file is part of the klaytn library.
+//
+// The klaytn library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The klaytn library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the klaytn library. If not, see <http://www.gnu.org/licenses/>.
+
+// SPDX-License-Identifier: LGPL-3.0-only
+pragma solidity ^0.8.18;
+
+import "./IKIP113.sol";
+import "./IAddressBook.sol";
+import "../lib/Initializable.sol";
+import "../lib/UUPSUpgradeable.sol";
+import "../lib/OwnableUpgradeable.sol";
+
+contract KIP113 is Initializable, UUPSUpgradeable, OwnableUpgradeable, IKIP113 {
```

Comment:
@hyeonLewis If we change the order of superclasses, does the storage layout change?

Reply:
Yes, it matters to storage layout.